resources/js/components/beatmapset-event.tsx

```diff
@@ -114,11 +114,11 @@ export default class BeatmapsetEvent extends React.PureComponent<Props> {
   }
 
   private content() {
-    let discussionLink: React.ReactChild = '';
-    let discussionUserLink: React.ReactChild = '[unknown user]';
-    let text: React.ReactChild = '';
+    let discussionLink: React.ReactElement | string = '';
+    let discussionUserLink: React.ReactElement | string = '[unknown user]';
+    let text: React.ReactElement | string = '';
     let url = '';
-    let user: React.ReactChild | undefined;
+    let user: React.ReactElement | string | undefined;
```

Comment:
might as well just use `React.ReactNode`? That's what's these variables end up being used for later in this function anyway